Readme.md

```diff
@@ -0,0 +1,147 @@
+README.md
+One attachment
+• Scanned by Gmail
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 MIRANJA TRACY <a name="about-project"></a>
+
+**MIRANJA TRACY hello world** is a webpage That i have created to test whether i grasp steps i learnt.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [Live Demo Link](https://github.com/Tracy-miranja/Myfirst-repo)
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- GETTING STARTED -->
+ 
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+ 
+### Prerequisites
+ 
+In order to run this project you need:
+ 
+- A computer.
+- Internet access.
+- A modern web browser.
+ 
+### Setup
+ 
+Clone this repository to your desired folder:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:Tracy-miranja/Tracy-miranja/hello-microverse.git
+```
+ 
+### Install
+ 
+Install this project with:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:Tracy-miranja/Tracy-miranja/hello-microverse.git
```

Comment:
- [ OPTIONAL] Great work adding the closing ticks as requested by the first reviewer. However, I noticed that you repeated the same commands twice. I would advise that you do not repeat the commands. 👍 

![image](https://user-images.githubusercontent.com/77106962/211668592-02d0ac38-4609-4e2f-87e8-33977ebf52fc.png)